dtfs-central-api/src/repositories/tfm-deals-repo/tfm-deal-cancellation.repo.ts

```diff
@@ -32,7 +24,7 @@ export class TfmDealCancellationRepo {
     }
 
     if (!matchingDeal.tfm.cancellation) {
-      throw new DealCancellationNotFoundError(dealId.toString());
+      return {};
```

Comment:
Why do we want to return an object here - is the cancellation object not created when a deal reaches TFM (and therefore all deals should have a cancellation object)?